content/en/docs/architecture/release-gating.md

```diff
@@ -67,88 +36,215 @@ Good candidates include:
 
 After at least one sprint of running the job, and success rate is at least 70%, the job will be considered stable enough
 to be considered for inclusion to the nightly informing jobs.
 
-To become an informing
-job, [file a ticket in the TRT Jira project](https://issues.redhat.com/secure/CreateIssue.jspa?pid=12323832&issuetype=17),
+To become an informing job, [file a ticket in the TRT Jira project](https://issues.redhat.com/secure/CreateIssue.jspa?pid=12323832&issuetype=17),
 containing:
 
 - The names of the jobs to become informing
+- Justification for payload blocking over component readiness
 - Links to the job history in [Sippy](https://sippy.dptools.openshift.org/) showing its historical pass rate
 - A channel to send Slack alerts, and the threshold to alert on (minimum of 70%)
 - A method for escalation of breakages to the team (jira, slack, e-mail, etc)
 - A manager backstop to contact if the job fails for a prolonged period
 
 Jobs that fail for an extended period of time without an expectation of return to stability within a reasonable timeframe
-(~1 sprint), should be evaluated for removal as an informing job.
+(~1 sprint), should be evaluated for removal as an informing job. Jobs that have 
+remained informing for 2 releases without going blocking should be removed from 
+informing status (but may continue to run as scheduled periodics). 
+
+#### Extract blocking signal
+
+An optional step along the way to become blocking would be to extract partial blocking 
+signal from informing jobs using blocking analysis jobs.
+
+We are able to analyze a set of informers matching specific criteria, and ensure that a particular
+part of the job passes. We have two of them today, `install-analysis-all` and `overall-analysis-all`. Install analysis
+ensures that we get at least one successful install across groups of jobs. For example, we can ensure we get at least one
+good install across each platform, each CNI, each IP stack, etc. `overall-analysis-all` ensures that the test step succeeds,
+effectively acting as a proxy for "the job passes entirely." For example, we require that techpreview jobs pass all serial
+and parallel conformance tests at least once on any platform.
+
 
 #### Becoming Blocking
 
 After at least 2 sprints of informing status, you may request the job become blocking by again
 [filing a ticket in the TRT Jira project](https://issues.redhat.com/secure/CreateIssue.jspa?pid=12323832&issuetype=17).
 
-Blocking jobs come with the responsibility of being responsive to breakages.  Generally, TRT expects a team that requests
+Blocking jobs come with the responsibility of being responsive to breakages. Generally, TRT expects a team that requests
 a blocking job to respond to a breakage within 4 hours during their normal working hours only. In exchange for this agreement,
 TRT will monitor the jobs, alert on them, and revert changes in the product that caused the breakage.
 
+#### Once with retries, or aggregated?
```

Comment:
This section is fine, but I think new instances of aggregated jobs will be rare.  I'd be interested in vsphere and HCP I suppose.

I suggest that we try to derive more value from our informing jobs.

Reply:
> I suggest that we try to derive more value from our informing jobs.

I'm trying to avoid creation of forever informing jobs at all; they should only be created unless they have a path to fully blocking, otherwise the signal belongs in component readiness.  There's too many low value informers that make the release controller a mess.